test/services/account/services/jwtTimerService.test.js

```diff
@@ -0,0 +1,126 @@
+const assert = require('assert');
+const { decode } = require('jsonwebtoken');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['decode' is assigned a value but never used. (no-unused-vars)](https://app.codacy.com/app/schul-cloud/schulcloud-server/pullRequest?prid=4141092)